openwisp_firmware_upgrader/settings.py

```diff
@@ -6,3 +6,6 @@
 UPGRADERS_MAP = getattr(settings, 'OPENWISP_FIRMWARE_UPGRADERS_MAP', {
     conn_settings.DEFAULT_UPDATE_STRATEGIES[0][0]: 'openwisp_firmware_upgrader.upgraders.openwrt.OpenWrt'
 })
+
+PRIVATE_STORAGE_CLASS = 'openwisp_firwmare_upgrader.private_storage.storage.OpenWISPPrivateFileSystemStorage'
+OPENWISP_FIRMWARE_UPGRADER_MAX_FILE_SIZE = getattr(settings, 'OPENWISP_FIRMWARE_UPGRADER_MAX_FILE_SIZE', None)
```

Comment:
can you default this to 30 MB please?

Reply:
another thing, for consistency it should be:

`MAX_FILE_SIZE = getattr(settings, 'OPENWISP_FIRMWARE_UPGRADER_MAX_FILE_SIZE', None)`